server/planning/content_profiles/planning_types_async_service.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2021 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
```

Comment:
Let's get rid of this. The license is already in the repository and the license link here is broken anyways.